components/eamxx/src/physics/mam/eamxx_mam_microphysics_process_interface.hpp

```diff
@@ -5,24 +5,14 @@
 #include <share/atm_process/atmosphere_process.hpp>
 #include <share/util/scream_common_physics_functions.hpp>
 
-#include "impl/mam4_amicphys.cpp" // mam4xx top-level microphysics function(s)
+// For reading files
```

Comment:
Is this header needed in the hpp? My guess is "no", in which case I'd include it in the cpp file

Reply:
Yes, the header is needed in the .hpp file for the definition of variables that are required by the reader.